docs/integrations/webhooks/sync-data.mdx

```diff
@@ -105,213 +114,144 @@ This guide can be adapted to listen for any Clerk event.
     ```
   </CodeBlockTabs>
 
-  ### Create the endpoint in your application
+  ### Create the endpoint
 
-  Create a Route Handler that uses `svix` to verify the webhook signature and that receives the webhook's payload.
+  Set up a Route Handler that uses `svix` to verify the webhook and process the payload.
 
-  For the sake of this guide, you will only log the payload to the console. In a real world application, you would use the payload to trigger some action. For example, you are listening for the `user.updated` event, so you could perform a database `update` or `upsert` to update the user's details.
+  For this guide, the payload will be logged to the console. In a real app, you'd use the payload to trigger an action. For example, if listening for the `user.created` event, you might perform a database `update` or `upsert` to update the user's details.
 
-  Your webhook will need to return either an error code, or a success code, like `200` or `201`. If it returns an error code, the webhook will reflect that in the Dashboard log and it will [retry](/docs/integrations/webhooks/overview#retry). When the webhook returns a success code, there will be no retries and the webhook will show a success status in the Dashboard.
+  The webhook must return either an [error code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses) such as `405` or `403` or a [success code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses), such as `200` or `201`. An error code will appear in the Clerk Dashboard log and trigger a [retry](/docs/integrations/webhooks/overview#retry). A success code will stop further retries and show a success status in the Clerk Dashboard log.
```

Comment:
This should probably be more like:

If the route handler returns a 2xx code that will mark the even as successful and stop retries.
If the route handler returns a 4xx or 5xx code or no code at all, then the webhook event will be retried.

Basically a 2xx/success code stops retries and anything else does not and is considered a failure.